tests/components/synology_dsm/test_config_flow.py

```diff
@@ -274,29 +274,19 @@ async def test_reauth(hass: HomeAssistant, service: MagicMock):
 
         result = await hass.config_entries.flow.async_init(
             DOMAIN,
-            context={
-                "source": SOURCE_REAUTH,
-                "data": {
-                    CONF_HOST: HOST,
-                    CONF_USERNAME: USERNAME,
-                    CONF_PASSWORD: PASSWORD,
-                },
+            context={"source": SOURCE_REAUTH},
```

Comment:
Möte that there will be more dwtoin the context set by default when raising `ConfigEntryAuthFailed`, ie config entry id and config entry unique_id.

Reply:
thanks for that hint, have added missing details